ui/src/main/resources/FileManagerCode/Pagination.xml

```diff
@@ -223,6 +238,15 @@
           var offset = newValues[1] || 0;
           var limit = newValues[2] || 15;
 
+          if (!scope.filesLimitOptions &amp;&amp; newValues[2]) {
```

Comment:
What if ``filesLimitOptions`` is defined, but the specified limit is not included in the list?

Reply:
The list will remain the same, but the filter will be applied in the LiveTable (the selected option will become blank).

On a new page load (with no query string), the `if` executes and adds 15 to the list. Afterwards, the only way to change the files per page to a custom amount is by modifying the query string (i.e. `l=44`, which isn't added to the list). Refreshing the page does add the new value to the options.

I initially added the `if` to not recompute the options every time one of `totalCount`, `offset` or `limit` has a value change.